.github/workflows/pr.yml

```diff
@@ -40,9 +40,9 @@ jobs:
           dart run realm_dart deploy-apps \
             --baas-url https://realm-dev.mongodb.com \
```

Comment:
Is the url not different for QA?

Reply:
Ah... forgot that I had hardcoded it - will update